source/uwp/Renderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -704,24 +704,26 @@ namespace AdaptiveNamespace
                 WeakRef weakParent;
                 THROW_IF_FAILED(localParentElement.AsWeak(&weakParent));
 
-                WeakRef weakEllipse;
-                THROW_IF_FAILED(ellipseAsUIElement.AsWeak(&weakEllipse));
                 THROW_IF_FAILED(brushAsImageBrush->add_ImageOpened(
```

Comment:
>add_ImageOpened [](start = 51, length = 15)

Would switching to wpp/winrt help simplify this eventregistration pattern? Do we actually want the callback to be invoked if the parent object teh weakref in the lambda is pointing to, gets destructed?

Reply:
The registration pattern *would* look nicer in cpp/winrt. However, we aren't currently using that pattern anywhere in our codebase, so this would be the first instance. Since this is a regression fix, I didn't want to hold up shipping it to figure out the lifetime patterns or change a bunch of code...

---
In reply to: [369931944](https://github.com/microsoft/AdaptiveCards/pull/3739#discussion_r369931944) [](ancestors = 369931944)

Reply:
Also, I didn't really dig in to why `weakEllipse` was resolving as `nullptr`.

---
In reply to: [370266579](https://github.com/microsoft/AdaptiveCards/pull/3739#discussion_r370266579) [](ancestors = 370266579,369931944)